File: 2026/08-August/25-Tuesday/solution.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int MissingMultiple(int[] nums, int k) {
        HashSet<int> set = new HashSet<int>(nums);

        for (int i = 1; ; i++) {
            int x = i * k;

            if (!set.Contains(x)) {
                return x;
            }
        }
    }
}

File: 2026/08-August/25-Tuesday/solution.java
class Solution {
    public int missingMultiple(int[] nums, int k) {
        HashSet<Integer> set = new HashSet<>();

        for (int num : nums) {
            set.add(num);
        }

        for (int i = 1; ; i++) {
            int x = i * k;

            if (!set.contains(x)) {
                return x;
            }
        }
    }
}
